src/main/java/bio/terra/service/snapshot/SnapshotService.java

```diff
@@ -585,16 +552,16 @@ public Snapshot makeSnapshotFromSnapshotRequest(SnapshotRequestModel snapshotReq
                             "This dataset does not have an asset specification with name: "
                                 + assetName));
         snapshotSource.assetSpecification(queryAssetSpecification);
-        // TODO this is wrong? why dont we just pass the assetSpecification?
+        // TODO this is wrong? why don't we just pass the assetSpecification?
         conjureSnapshotTablesFromAsset(
             snapshotSource.getAssetSpecification(), snapshot, snapshotSource);
-        break;
-      case BYROWID:
+      }
+      case BYROWID -> {
         SnapshotRequestRowIdModel requestRowIdModel = requestContents.getRowIdSpec();
         conjureSnapshotTablesFromRowIds(requestRowIdModel, snapshot, snapshotSource);
-        break;
-      default:
-        throw new InvalidSnapshotException("Snapshot does not have required mode information");
+      }
+      default -> throw new InvalidSnapshotException(
+          "Snapshot does not have required mode information");
```

Comment:
Since this `switch` is exhaustive I would omit `default`. If a new element is added to the `enum` this code will fail to compile, which won't happen if `default` is left in.